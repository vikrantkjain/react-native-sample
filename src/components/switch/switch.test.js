import ToggleSwitch from './toggleSwitch';
import render from '@testing-library/react-native';
import renderer from 'react-test-renderer';

test('stop in the middle of animation', () => {
  const innerCircleTrackStyle = {off: 'red', on: '#fff'};
  const backgroundTrack = {off: '#fff', on: 'green'};

  const tree = renderer
    .create(
      <ToggleSwitch
        innerCircleTrack={innerCircleTrackStyle}
        backgroundTrack={backgroundTrack}
      />,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
