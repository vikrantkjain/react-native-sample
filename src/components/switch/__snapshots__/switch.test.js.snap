// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`stop in the middle of animation 1`] = `
<View
  collapsable={false}
  style={
    [
      {
        "justifyContent": "center",
      },
      {
        "borderRadius": 11.25,
        "height": 22.5,
        "width": 50,
      },
      {
        "backgroundColor": undefined,
      },
    ]
  }
  testID="TEST_VIEW"
>
  <View
    style={
      [
        {
          "backgroundColor": "#fff",
          "position": "absolute",
          "zIndex": 1,
        },
        {
          "borderRadius": 11.25,
          "height": 22.5,
          "width": 22.5,
        },
        {
          "backgroundColor": undefined,
          "transform": [
            {
              "translateX": 0,
            },
          ],
        },
      ]
    }
  />
</View>
`;
